README.md

```diff
@@ -0,0 +1,96 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="micro.png" alt="logo" width="500"  height="auto" />
+  <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is a git and github training project.
+
+## 🛠 Built With <a name="built-with"></a>
```

Comment:
- [ ] [OPTIONAL] I notice that your commit messages are not very professional. Please read through [this article](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) and I believe you will learn best practices.

👇🏾 
![image](https://user-images.githubusercontent.com/84156714/220065039-22821b1f-e62b-4ce7-9b78-3fdd2fafa1e0.png)